### PR TITLE
Add vagrant files (by clsung https://github.com/clsung/cython_vagrant).

### PR DESCRIPTION
**Pull request by yungyuc**
Should we manage vagrant files in this repository?

### CODE REVIEW

**Comment by clsung** [comment]
Sure, after we freeze the very first version with ipython[32] support

**Comment by yungyuc** [comment]
@clsung Does that mean developing VM files in an additional repository is a better idea?

**Comment by yungyuc** [comment]
@clsung How do you want to proceed with the vagrant files?

**Comment by yungyuc** [comment]
@clsung Any idea about how we should do with this PR?

**Comment by tjwei** [comment]
the git url in cython_sample.sls should be  https://github.com/yungyuc/cythonup.git instead of https://github.com/clsung/cythonup.git

**Comment by yungyuc** [comment]
We just got http://pycourse.org/ and https://github.com/pycourse set up.  If we want to move this repository there I can do it at your commands.

**Comment by tjwei** [comment]
there are still a few packages missing in the vm. numpy is not installed for example. Also it would be handy for me if pypy is also available. But anyway, we shall decide whether the vagrant is going merge into the master of this repo. Either way is fine by me and make sense, but current situation makes it difficult to fix issues for the vm.

**Comment by yungyuc** [comment]
Agree that we should have a way to collaborate on the VM.  It doesn't
matter to me whether it's in the cythonup repository or use another one,
though I don't see obvious reason not merging this PR.  I hope @clsung can
make the final decision.  We are in 10 days to the show.

On Wed, Jul 22, 2015 at 2:31 PM, tjwei notifications@github.com wrote:

> there are still a few packages missing in the vm. numpy is not installed
> for example. Also it would be handy for me if pypy is also available. But
> anyway, we shall decide whether the vagrant is going merge into the master
> of this repo. Either way is fine by me and make sense, but current
> situation makes it difficult to fix issues for the vm.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/yungyuc/cythonup/pull/1#issuecomment-123576844.

## 

Yung-Yu Chen
http://yyc.solvcon.net/
+886 (99) 129 4763 (no sms)

File: cython_vagrant/salt/roots/salt/cython_sample.sls [hunk 1]
```diff
@@ -0,0 +1,5 @@
+https://github.com/clsung/cythonup.git:
```
Comment (clsung) [inline cython_vagrant/salt/roots/salt/cython_sample.sls:1]: we need to change this to https://github.com/yungyuc/cythonup.

Comment (yungyuc) [inline cython_vagrant/salt/roots/salt/cython_sample.sls:1]: @clsung You haven't accepted PR #1 so that no one can change it in the blessed repo.

**Pull request merged**